### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio

### ISSUE THREAD

**Issue by ErriGarcia** [Coding-Coach/coding-coach-front-end#433]
**Description**:
Low-contrast text poses difficulties for a significant number of users, impeding their ability to read content effectively. In an effort to address this accessibility concern, we attempted to set a different brand color to undergo WCAG color tests. However, the current brand color, #69D5B1, fails to meet the required contrast ratio of 4.5:1 (or 3:1 for large text), achieving only a contrast ratio of 1.78:1.

<img width="457" alt="Screenshot 2023-06-21 alle 06 02 43" src="https://github.com/Coding-Coach/coding-coach/assets/113127647/677e4f4e-b272-45bb-817c-4ae6ec5960f5">

**Proposed Solution**:
We recommend using a different color, #255645, which boasts a contrast ratio of 8.41:1. This alternative color would substantially enhance the legibility and accessibility of the text, ensuring that it can be easily read by all users.

![image](https://github.com/Coding-Coach/coding-coach/assets/113127647/3b250bb4-48a1-41ed-9098-8d65760b83b2)

Steps to Reproduce:

- [ ] - Navigate to the webpage containing low-contrast text.
- [ ] - Inspect the current brand color, #69D5B1, and its associated contrast ratio.
- [ ] - Observe the inadequate contrast ratio of 1.78:1.

Expected Result:
Text should have a contrast ratio of at least 4.5:1 (or 3:1 for large text) to ensure optimal legibility and accessibility.

Additional Information:
We are committed to assisting with the necessary changes to improve accessibility. Please let us know how we can be of further assistance in resolving this issue and ensuring an inclusive user experience.

Thank you for your attention to this matter.

@ttkapostol @ErriGarcia @NataliaBlanco
